src/components/MoodForm/MoodForm.tsx

```diff
@@ -0,0 +1,127 @@
+import './MoodForm.css';
+import { useState } from 'react';
+import { useMutation } from '@apollo/client';
+import { SUBMIT_MOOD } from '../../utils/graph_mutations';
+
+const MoodForm = () => {
+  const [ mood, setMood ] = useState('')
+  const [ description, setDescription ] = useState('')
+  const [ validateForm, setValidateForm ] = useState(true)
```

Comment:
I think the name of this variable could be better, but I couldn't think of something different. Any suggestions welcomed.